sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -667,17 +667,13 @@ class ExpressionParserSuite extends AnalysisTest {
     }
 
     // Unknown FROM TO intervals
-    intercept("interval 10 month to second",
+    intercept("interval '10' month to second",
```

Comment:
from-to unit can only handle string values (`'1-2' year to month`, `'1 12' day to hour`).

Reply:
If you do write `interval 10 month to second`, then the parser thinks `interval 10 month to` is a valid expression (interva literal with alias) and `second` should be removed.